game_board.py

```diff
@@ -1,126 +1,64 @@
 import pyglet
+import resources
+from pyglet.window import mouse
 
+path = '/Users/siczek/Python/final_game/git_repos/resources/'
```

Comment:
Constant names should be written in ALL_CAPS.

`/Users/siczek/Python/final_game/git_repos/` should not be hardcoded. You can use [os.getcwd()](https://docs.python.org/3.7/library/os.html#process-parameters) for this, I think (i.e. `path = f'{os.getcwd()}{os.sep}resources`)

Reply:
OMG, I just saw it 😳 I will change this.